test/cases/ostree.sh

```diff
@@ -773,6 +799,437 @@ sudo ansible-playbook -v -i "${TEMPDIR}"/inventory \
     /usr/share/tests/osbuild-composer/ansible/check_ostree.yaml || RESULTS=0
 check_result
 
+greenprint "🧹 Clean up VM"
+if [[ $(sudo virsh domstate "${IMAGE_KEY}") == "running" ]]; then
+    sudo virsh destroy "${IMAGE_KEY}"
+fi
+sudo virsh undefine "${IMAGE_KEY}" --nvram
+sudo virsh vol-delete --pool images "$LIBVIRT_IMAGE_PATH"
+
+# The following pulp test is only for rhel-9.3 and later
+if [[ "${SKIP_PULP_TEST}" == "true" ]]; then
+    greenprint "📋 Current OS does not support pulp, skip pulp test and exit"
+    clean_up
+    exit 0
+fi
+
+##################################################
+##
+## ostree upload commit to pulp
+##
+##################################################
+
+# Setup pulp server
+greenprint "📄 Setup pulp server with one container"
+mkdir -p settings pulp_storage pgsql containers
+echo "CONTENT_ORIGIN='http://$(hostname):8080'
+ANSIBLE_API_HOSTNAME='http://$(hostname):8080'
+ANSIBLE_CONTENT_HOSTNAME='http://$(hostname):8080/pulp/content'
+CACHE_ENABLED=True" >> settings/settings.py
+sudo podman run --detach \
+            --publish 9090:80 \
+            --name pulp \
+            --volume "$(pwd)/settings":/etc/pulp:Z \
+            --volume "$(pwd)/pulp_storage":/var/lib/pulp:Z \
+            --volume "$(pwd)/pgsql":/var/lib/pgsql:Z \
+            --volume "$(pwd)/containers":/var/lib/containers:Z \
+            --device /dev/fuse \
+            docker.io/pulp/pulp:latest
```

Comment:
@achilleas-k @yih-redhat since the pulp-ostree issue has been closed and the PR merged, the changes are available in their nightly build of the container. I've tested it locally and it appears to work now

Reply:
@kingsleyzissou Thanks, let's wait and see the CI test result.

Reply:
Seems not fixed in the latest container docker.io/pulp/pulp:latest

........Error: Task /pulp/api/v3/tasks/018a82b0-4ca7-73da-a347-3dfdceaa40fe/ failed: 'The parent commit '37b860943b1b39f1cbade7e0f71016c8e64b34c96510d0b73981a567ac4514d2' could not be loaded'

Reply:
Yeah when I was testing it looks like only the `nightly` has the fix

Reply:
I can change to nightly, to make the test pass. But currently the CI test is not started because of RPM build failed for some os, can rebase fix these failures?

Reply:
Clicked the rebase.  Let's see.